fashrevwall/models/models.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+
+class Tweet(models.Model):
+    text = models.CharField(max_length=200)
```

Comment:
OK, thought it could be useful later but maybe just clutters the DB. Removing :)